.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules
 package-lock.json
 data.csv
-newrelic.js
-newrelic_agent.log
+config.js
```

Comment:
You could add `*.DS_Store` and then `git rm --cached *.DS_store` to remove those extraneous Mac systems files from your project. https://alvinalexander.com/git/git-rm-file-git-repository-dont-track-accidentally-added